config/app.config

```diff
@@ -27,9 +27,8 @@ whitelisted_warnings: ["ANS1809W", "ANS2042W", "ANS2250W"]
 
 # Exclude filters (different on biotank and Irma)
 #
-# Dirs and file extensions to exclude from the _archive copy of the runfolder/project
+# Dirs to exclude from the _archive copy of the runfolder/project
 exclude_dirs: ["Data", "Thumbnail_Images"]
```

Comment:
Oh, I had forgotten about `exclude_dirs`, I think this should be made into a parameter too, since bcl files always are located in the `Data` folder.

Reply:
Fixed